Sources/CairoGraphics/image/CairoImage.swift

```diff
@@ -56,12 +68,20 @@ public final class CairoImage: BufferedImage {
     }
 
     public convenience init(width: Int, height: Int) throws {
-        let surface = try Surface.Image(format: .argb32, width: width, height: height)
-        self.init(rawSurface: surface)
+        try self.init(width: width, height: height, format: .rgba32)
     }
 
     public convenience init(size: Vec2<Int>) throws {
-        try self.init(width: size.x, height: size.y)
+        try self.init(size: size, format: .rgba32)
+    }
+
+    public convenience init(width: Int, height: Int, format: PixelFormat) throws {
+        let surface = try Surface.Image(format: format.imageFormat, width: width, height: height)
+        self.init(rawSurface: surface)
+    }
+
+    public convenience init(size: Vec2<Int>, format: PixelFormat) throws {
+        try self.init(width: size.x, height: size.y, format: format)
     }
```

Comment:
Thinking about it, we could probably move all of these convenience initializers (except `init(width:height:format:)` into a protocol extension on `BufferedImage`).

Reply:
Good idea. Done.